test_analysis/fake_data_stan_analysis_termite.py

```diff
@@ -0,0 +1,513 @@
+#
+# fake_data_stan_analysis.py
+# Author: T. E. Weiss
+# Date modified: June 2, 2020
+#
+# This script generates fake data, then analyzes the data in Stan to infer posteriors.
+# Pathnames configured for running from: termite/phase2_main_scripts/
+#
+
+"""
+To-do:
+    - In FakeExperimentEnsemble, add:
+        1. Tracking of convergence issues, so that a summary of problems can be saved/printed
+        2. An option to parallelize with slurm instead of multiprocessing
+    - Run morpho processor for ensemble-analysis plotting, once it has been tested sufficiently
```

Comment:
add example for how to use it and note that a script like this (but in termite) was used for the official Phase II bayesian analysis.